### PR TITLE
groonga: add HEAD support

### PR DESCRIPTION
**Pull request by cosmo0920**
Some of Groonga users want to try new features as soon as possible.
It's reasonable to support build with `--HEAD` option.
How do you think about it?

### CODE REVIEW

File: Library/Formula/groonga.rb [hunk 1]
```diff
@@ -32,10 +39,12 @@ def install
       --without-libstemmer
     ]
 
+    args << "--with-ruby" if build.head?
     args << "--enable-benchmark" if build.with? "benchmark"
     args << "--with-mecab" if build.with? "mecab"
     args << "--with-lz4" if build.with? "lz4"
 
+    system "./autogen.sh" if build.head?
```
Comment (DomT4) [inline Library/Formula/groonga.rb]: You could put this in one block :)

```ruby
if build.head?
 args << "--with-ruby"
 system "./autogen.sh"
end
```

Comment (cosmo0920) [inline Library/Formula/groonga.rb]: Thanks, I've fixed it!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/6e43e9a35174000aac3d2275a561a0f43f98bb2a. Thank you once more @cosmo0920! :smiley_cat:

**Comment by cosmo0920** [comment]
Thank you for your advice and quick merging! :yum:

**Pull request closed without merging**